index.html

```diff
@@ -19,9 +19,11 @@ <h1>Quiz App</h1>
     <main class="app-grid__main">
       <section class="card">
         <div class="bookmark">
```

Comment:
I don't think you need the `div`-element here. You also don't need the anchor element. You should be able to style the icon directly by giving it additional classes.